odbc/src/api/data.rs

```diff
@@ -1102,7 +1095,7 @@ pub mod i16_len {
         if output_ptr.is_null() {
             return SqlReturn::SUCCESS_WITH_INFO;
         }
-        copy_nonoverlapping(data as *const _, output_ptr as *mut _, 1);
```

Comment:
since these `set_output_fixed_date` functions only copy 1 thing, I'm a little confused on what the point of them is. They seem very limited. Also, I noticed that sometimes they are used to copy to buffers much bigger than 1 `sizeof::<T>()` bytes which I was wondering about.